packages/tempus/test/BackwardEuler/Tempus_BackwardEulerTest.cpp

```diff
@@ -17,11 +18,13 @@
 
 #include "../TestModels/SinCosModel.hpp"
 #include "../TestModels/CDR_Model.hpp"
+#include "../TestModels/CDR_Model_Tpetra.hpp"
 #include "../TestModels/VanDerPolModel.hpp"
 #include "../TestUtils/Tempus_ConvergenceTestUtils.hpp"
 
 #include "Stratimikos_DefaultLinearSolverBuilder.hpp"
 #include "Thyra_LinearOpWithSolveFactoryHelpers.hpp"
+#include "Tpetra_Core.hpp"
 
 #ifdef Tempus_ENABLE_MPI
 #include "Epetra_MpiComm.h"
```

Comment:
Do you still need this include?

Reply:
Yes, this file runs both Epetra(old) and Tpetra(new) versions of CDR_Model.
Unless the SDR will want to get rid of epetra version, I'm keeping the old one for backward compability